webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -110,6 +168,23 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           ]}>
           <Input maxLength={80} />
         </Form.Item>
+        {canModifyBindings && Loadable.getOrElse([], resourceManagers).length > 0 && (
+          <>
+            <Divider />
+            Namespace Bindings:
+            <Body inactive>
+              Note: If you leave the Namespace name blank, the workspace will be bound to the
+              default Namespace configured in the Master Config.
+            </Body>
+            <>
+              {Loadable.getOrElse([], resourceManagers).map((name) => (
+                <Form.Item key={name} label={name} name={name}>
```

Comment:
re: above -- we can nest items by passing the path to the field as an array of keys:
```suggestion
                <Form.Item key={name} label={name} name={['bindings', name]}>
```